util/options_helper.h

```diff
@@ -688,6 +688,18 @@ static std::unordered_map<std::string, CompactionStyle>
         {"kCompactionStyleFIFO", kCompactionStyleFIFO},
         {"kCompactionStyleNone", kCompactionStyleNone}};
 
+static std::unordered_map<char, std::string> compaction_style_logging_map = {
```

Comment:
unordered_map does not support enum as a key well. Unless we have something like: (http://stackoverflow.com/questions/18837857/cant-use-enum-class-as-unordered-map-key). Is that what we want?